packages/frontend/src/app/submissions/[id]/components/LazySubmissionDetail.tsx

```diff
@@ -0,0 +1,104 @@
+import Image from 'next/image'
+import { Text2xlBold, TextSmBold } from '@/common/components/Typography'
+import { Submission } from '@/common/interfaces'
+import { Label } from '@/common/components/Label'
+
+export function LazySubmissionDetail({
```

Comment:
I'm not sure about the name `Lazy...`. Do we have a non-lazy variant? Under the term lazy I would imagine a component that's rendered and then lazyly load its content, but I assume that's not the case here.